src/OpenTracing/Exceptions/SpanAlreadyFinished.php

```diff
@@ -16,7 +16,11 @@
  */
 final class SpanAlreadyFinished extends DomainException
 {
-    public static function create(Span $span)
+    /**
+     * @param Span $span
+     * @return SpanAlreadyFinished
+     */
+    public static function forSpan(Span $span)
```

Comment:
there have been discussions about explicitly making Finish() method idempotent. In the best case the implementation will be smart enough ti ignore extra calls to Finish(), perhaps log a warning, and in the worst case it can throw an exception which it can internally define (throwing exception in this case is just bad practice imo). So does the API really need this exception?